contracts/wrapper/NameWrapper.sol

```diff
@@ -1096,4 +1182,12 @@ contract NameWrapper is
             ownerOf(uint256(node)) != address(0) &&
             ens.owner(node) == address(this);
     }
+
+    function _isDotEthAndIsNotExpired(
```

Comment:
Shouldn't this be something like `isNotDotEthOrIsNotExpired`? Can you elaborate on why we don't need to check expirations for non-.eth names?

Reply:
This is probably a bad name. The reason this check exists is because we wanted to set the expiry to .eth expiry + grace period. However we do not want to allow them to keep operating on the name, so for the canModifyName checks we subtracted GRACE_PERIOD from the expiry.

Reply:
The name definitely needs changing; it doesn't do what it says right now.

Doesn't this check still allow operating on expired names as long as they're not .eth 2LDs?

Reply:
No it doesn't. Because when a name is truly expired, the owner is zero'd out by getData(). However .eth are special now because their expiry now includes the grace period